release/models/lldp/openconfig-lldp-types.yang

```diff
@@ -302,5 +308,30 @@ module openconfig-lldp-types {
       "IEEE 802.1AB LLDP MIB";
   }
 
+  typedef mgmt-interface-number-subtype {
+    type enumeration {
+      enum UNKNOWN {
+        value 1;
+        description
+          "Interface is not known";
+      }
+      enum IFINDEX {
+        value 2;
+        description
+          "Interface is based on the IF-MIB ifIndex";
+      }
+      enum SYSTEM_PORT_NUMBER {
+        value 3;
+        description
+          "Interface is based on the system port number";
```

Comment:
If you prefer, this is the exact text in the IEEE spec:

`Represents interface identifier based on the system port numbering convention.`